src/PKSim.UI.Starter/ApplicationStartup.cs

```diff
@@ -54,24 +74,86 @@ private static IContainer initializeForStartup(IShell shell)
          container.Register<IApplicationController, ApplicationController>(LifeStyle.Singleton);
          container.Register<ICoreWorkspace, OSPSuite.Core.IWorkspace, IWorkspace, Workspace>(LifeStyle.Singleton);
 
-         // registerImplementationsFromCore(container);
          registerMinimalImplementations(container);
 
 
          using (container.OptimizeDependencyResolution())
          {
+            container.RegisterImplementationOf(shell as BaseShell);
             container.RegisterImplementationOf(shell);
+            container.RegisterImplementationOf(UserLookAndFeel.Default);
```

Comment:
@georgeDaskalakis simpler than we imagined

Reply:
@georgeDaskalakis true :)